test/mongodb.test.js

```diff
@@ -1375,6 +1375,24 @@ describe('mongodb connector', function() {
         _id: { type: geoDb.ObjectID, id: true },
         location: { type: GeoPoint, index: true },
       });
+
+      createPost = function(far) {
```

Comment:
move function out and share it with other tests `function createPost(isFar, coordinates) {`

where coordinates are `{lat: i, lng: j}`

then reuse this shared func in all tests.